gnd/src/main/java/com/google/android/gnd/repository/DataRepository.java

```diff
@@ -121,13 +121,13 @@ private void onRemoteFeatureVectorChange(DatastoreEvent<Feature> event) {
                     .orElse(Single.error(new DocumentNotFoundException())));
   }
 
-  public Observable<Resource<Record>> getRecordDetails(
+  public Flowable<Resource<Record>> getRecordDetails(
```

Comment:
As of now, the record details viewmodel is the only place this method is called, so we may as well just have it return a Flowable.

Reply:
I think this makes sense semantically; iiuc Flowables are for cases where control the rate of emissions (e.g., database queries), whereas Observables are for cases where we don't (e.g., button clicks). We should eventually try to be consistent with what we use when, but I'm not yet sure of the right thing to do in the DataRepository. For now this seems ok!